src/layouts/CommunityDesktopShell.jsx

```
@@ -26,6 +58,8 @@ export default function CommunityDesktopShell({ children, headerContent = null,
   const defaultLeft = useMemo(() => {
     const params = new URLSearchParams(location.search);
     const tab = params.get('tab') || 'whatsGood';
+    const brand = params.get('brand') || '';
```

ESLint

[error] 'brand' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)